src/openapi/streaming/connection/connection.ts

```diff
@@ -0,0 +1,361 @@
+import log from './../../../log';
+import * as transportTypes from './transportTypes';
+import WebsocketTransport from './transport/websocket-transport';
+import SignalrTransport from './transport/signalr-transport';
+import SignalrCoreTransport from './transport/signalr-core-transport';
+import type { TransportTypes, ConnectionOptions } from '../types';
+import type {
+    StreamingTransportInterface,
+    ReceiveCallback,
+    StateChangeCallback,
+} from './types';
+
+type Callback = (...args: any[]) => unknown | void;

```

Comment:
why the unknown?

Reply:
Not needed, looks like a leftover from some fixes, will check if this type fits all usages.

Reply:
Fixed